pages/wizard/regions/task_step_bar.py

```diff
@@ -24,6 +24,22 @@
         'class': 'SetupType',
         'library': 'pages.wizard.rhev.setup_type',
     },
+    'Engine': {
+        'class': 'Engine',
+        'library': 'pages.wizard.rhev.engine',
+    },
+    'Hypervisor': {
+        'class': 'Hypervisor',
+        'library': 'pages.wizard.rhev.hypervisor',
+    },
+    'Configuration': {
```

Comment:
This is going to be a problem....we need to change my design somehow.   CloudForms.Configuration exists too, but we only put the menu text without  any of the path.   Don't change this....I'll be thinking about how we can fix this....probably we will all discuss.